ext/intl/formatter/formatter_main.c

```diff
@@ -104,6 +110,66 @@ PHP_METHOD( NumberFormatter, __construct )
 }
 /* }}} */
 
+/* {{{ proto void NumberFormatter::__wakeup()
+ * NumberFormatter object unserializer.
+ */
+PHP_METHOD( NumberFormatter, __wakeup )
+{
+		const char* locale = intl_locale_get_default();
```

Comment:
still uses default locale if serialization data do not contain locale value. Instead, it should fail.